notebooks/neural_networks/autoencoder.ipynb

```diff
@@ -7,17 +7,39 @@
     "# Using an auto encoder on MNIST handwritten digits."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial, we are going to learn how to compress handwritten digit images from MNIST dataset and reconstruct them by using [autoencoder](https://en.wikipedia.org/wiki/Autoencoder)　algorithm. The autoencoder model is mainly composed of ***encoder*** and ***decoder***. As you can see in the following diagram(*credit to this* [blog](https://blog.keras.io/building-autoencoders-in-keras.html)) of the model architecture, ***after the original input is fed through the encoder and decoder layer, the target output is expected to be the same as the input although there could be lost information actually after reconstruction***, which is the key point of autoencoder. It's a very basic and limited data compression algorithm but still good enough for practical applications such as data denoising and dimensionality reduction for data visualization."
```

Comment:
The second sentence seems missing article. "The autoencoder model is mainly composed of **a** encoder and **a** decoder"

Reply:
Thanks for pointing it our. Already fixed by the rebased commit below.